packages/shopify-codemod/README.md

```diff
@@ -55,6 +55,44 @@ a(() => {
 });
 ```
 
+### `global-assignment-to-default-export`
```

Comment:
You might want to mention that this is intended for use with commoner given that it uses the `expose` directive.